ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLocationType.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <v-card flat rounded id="mhr-home-location-type" class="mt-8 pa-8">
+    <v-row no-gutters class="pt-1">
+      <v-col cols="12" sm="2">
+        <label class="generic-label" :class="{'error-text': false}">Location Type</label>
+      </v-col>
+      <v-col cols="12" sm="10" class="mt-n1">
+        <v-radio-group
+          id="location-type--radio-options"
+          v-model="locationTypeOption"
+          class="mt-0 pr-1"
+          hide-details="true"
+          :disabled="isVerifyingPid"
+        >
+          <!-- Dealers / Manufacturers Lot -->
+          <v-row no-gutters>
+            <v-col>
+              <v-radio
+                id="lot-option"
+                class="home-type-radio"
+                label="Dealer's / Manufacturer's lot"
+                active-class="selected-radio"
+                :value="HomeLocationTypes.LOT"
+              />
+              <v-expand-transition>
+                <v-form
+                  v-if="locationTypeOption === HomeLocationTypes.LOT"
+                  ref="lotForm"
+                  v-model="isValidLot"
+                >
+                  <v-text-field
+                    filled
+                    id=""
```

Comment:
Maybe we can clean up empty IDs?

Reply:
Definitely, good catch. Will remove until required

Reply:
Removed! 
Also updated the Radio Btn id's 👍